src/assets/lib/canvasjs.react.js

```diff
@@ -0,0 +1,57 @@
+/*

```

Comment:
can we not import this library through npm?

Reply:
The documentation for this does not have set up instructions for npm. I did find an npm library for it, but that looks to be managed by a third party. Is it alright to import something using my method or is it not preferrable?

Reply:
I guess it is alright. Curious, why did you settle with canvas js if there is no npm package for it? Why not another charting lib?